### ISSUE TITLE
SG - mutation new target issues

### ISSUE THREAD

**Issue by ashleyindigital** [TheNexusMC-LLC/Issues#76]
When you mutate and they combat log it gives you a new player but the compass points to the old target's dead body

**Comment by Deamon5550** [comment]
I don't think this should give you a new target at all, wouldn't a more reasonable action be to simply refund your pass as your target 'died'.

**Comment by ashleyindigital** [comment]
I'd say get player's opinions on that cause a lot of the time, especially if it's someone well known, a person will combat log either right before they mutate or right after they mutate which would be rather annoying
